apps/ledger/src/credential-definition/credential-definition.service.ts

```diff
@@ -17,6 +17,8 @@ import { map } from 'rxjs/operators';
 import { OrgAgentType } from '@credebl/enum/enum';
 import { Cache } from 'cache-manager';
 import { CACHE_MANAGER } from '@nestjs/cache-manager';
+import { ICredDefDetails } from '@credebl/common/interfaces/cred-def.interface';
+import { sortValue } from 'apps/api-gateway/src/enum';
```

Comment:
We should add the gereric `enums` in libs folder

Reply:
Resolved